.github/workflows/marketplace-viz-ci-build.yml

```diff
@@ -0,0 +1,20 @@
+name: viz-ci-build
+on: [workflow_call]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repo checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          # Any higher and we need to update vizes to use webpack 5
```

Comment:
Do we need to create a bug for the vendors to do thing sometime in Q1 (i.e. use webpack 5)?

Reply:
Sure will file a low prio one.